dpgen2/superop/prep_run_lmp.py

```diff
@@ -129,6 +129,7 @@ def _prep_run_lmp(
     run_template_config = run_config.pop("template_config")
     prep_executor = init_executor(prep_config.pop("executor"))
     run_executor = init_executor(run_config.pop("executor"))
+    template_slice_config = run_config.pop("template_slice_config", {})
```

Comment:
Please add the argument definition for the `template_slice_config` to the [`step_config`](https://github.com/deepmodeling/dpgen2/blob/master/dpgen2/utils/step_config.py). 
An example on the `template_config` can be found [here](https://github.com/deepmodeling/dpgen2/blob/89f7c9590a8a9f914b0122a379448d59ad0ab0ca/dpgen2/utils/step_config.py#L106-L113) and [here](https://github.com/deepmodeling/dpgen2/blob/89f7c9590a8a9f914b0122a379448d59ad0ab0ca/dpgen2/utils/step_config.py#L68-L94)